test/state_machine/stm_tests.cpp

```diff
@@ -5,169 +5,164 @@
 namespace hyped::test {
 
 void testTransition(std::unique_ptr<state_machine::StateMachine> &stm,
```

Comment:
why is this a reference to a `unique_ptr`?

Reply:
I think this is from what I wrote last year, can just be changed